packages/titanium-loading-indicator/src/titanium-full-page-loading-indicator.ts

```diff
@@ -79,6 +79,7 @@ export class TitaniumFullPageLoadingIndicatorElement extends LitElement {
       this._animationTimer = window.setTimeout(() => {
         this.handleAnimationTimerEnd_();
         document.body.style.overflow = '';
+        this.opened = false;
```

Comment:
This is set to false already on line 76

Reply:
Yes. But a weird race condition with the render cycle makes it so sometimes it doesn't. This is just to ensure that it gets cleaned up at the end. On https://devpcms.leavitt.com/sales-expectation/10934/expectations with the loading veil as is, I was getting stuck on loading veil about 1 in 15 times I reload the page. With this, I wasn't able to get stuck in the veil.

Reply:
I think I see the issue here.  The the runNextAnimation is sometimes running after close was initiated.  

if we add to line 58 ``` if(this.closing) return; ``` , it should prevent this from occurring.